util/dvsim/Deploy.py

```diff
@@ -137,8 +137,8 @@ def _set_attrs(self):
         """
         self._extract_attrs(self.sim_cfg.__dict__)
 
-        # Enable GUI mode.
-        self.gui = self.sim_cfg.gui
+        # Enable GUI mode, also when GUI debug mode has been invoked.
+        self.gui = self.sim_cfg.gui or self.sim_cfg.gui_debug
```

Comment:
Couldn't we do things more easily if we ensured `gui_debug` implied `gui`?

Reply:
That's would be better indeed. I have added this in `FlowCfg.py` file:
```
        if self.gui_debug:
            self.gui = 1
```
Is that OK for you?

Reply:
That sounds like a good idea to me.